integrations/quick-start-guide.md

```diff
@@ -0,0 +1,48 @@
+# Integration Management Quickstart Guide
+In this quickstart guide, we will go over key features for integration management.
+
+## Integrations Page
+The integrations page is your home for adding and viewing integrations connected to your Aqueduct account. You can naviate to integrations page by clicking `Integrations` in the left menu bar. This page has two main sections: [add an integration](#add-an-integration) and [connected integrations](#connected-integrations).
+
+### Add an Integration
+To add an integration, simply click the icon for the integration you would like to connect. You would need to provide your service details together with your credentials. You could find more details [here](./adding-an-integration/) on connecting to s3, bigquery, and Google Cloud Storage (GCS).
+
+Once you added an integration, you can refer to [using integrations guide](./using-integrations/) to use the integration in Aqueduct SDK.
```

Comment:
Typo: Once you *have* added.

Reply:
Typo: to our guide on using integrations for more details on accessing integrations from the Aqueduct Python SDK.